### PR TITLE
chore(release): bump version to 0.13.4

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
+* **dev:** running command status message could become inconsistent ([0cd29f9a4](https://github.com/garden-io/garden/commit/0cd29f9a4))
+* **jib:** fix sha256 hashes for `mvnd` binaries ([0ea5f7857](https://github.com/garden-io/garden/commit/0ea5f7857))
+* **k8s:** update default Kaniko version ([6584369d4](https://github.com/garden-io/garden/commit/6584369d4))
+* **k8s:** invalid session name error with e.g. underscore in username/path ([62e8a96ad](https://github.com/garden-io/garden/commit/62e8a96ad))
+* **template-strings:** do not apply helper functions on unresolved string ([#4692](https://github.com/garden-io/garden/issues/4692)) ([0b47cccc6](https://github.com/garden-io/garden/commit/0b47cccc6))
+
+### Features
+
+* **openshift:** container Deploy action ([#4652](https://github.com/garden-io/garden/issues/4652)) ([1b448886a](https://github.com/garden-io/garden/commit/1b448886a))
+* **server:** add internal/experimental cmd for getting action statuses ([#4680](https://github.com/garden-io/garden/issues/4680)) ([b7471fbba](https://github.com/garden-io/garden/commit/b7471fbba))
```
Comment (shumailxyz) [inline CHANGELOG.md:21]: Do we want to include these in changelog?

Comment (Walther) [reply CHANGELOG.md:21]: I would personally recommend removing those, let's add a high-level changelog entry for experimental OpenShift support when all four action kinds are implemented

Comment (shumailxyz) [reply CHANGELOG.md:21]: I propose that we keep this in changelog.md file but skip adding this in the description of the release.

Comment (vvagaytsev) [reply CHANGELOG.md]: Removed unwanted entries in https://github.com/garden-io/garden/pull/4717/commits/95293224bf91ef9837908f71906c908a9e60aeea

**Pull request merged**